api/tests/mocha/services/connected-user-log.spec.js

```diff
@@ -0,0 +1,83 @@
+const chai = require('chai');
+const sinon = require('sinon');
+const moment = require('moment');
+const connectedUserLogService = require('../../../src/services/connected-user-log');
+const db = require('../../../src/db');
+
+describe('Connected Users Log service', () => {
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  describe('saveLog()', () => {
+    it('should save a log', () => {
+      const getStub = sinon.stub(db.medicLogs, 'get').returns(Promise.reject({ status: 404 }));
+      const putStub = sinon.stub(db.medicLogs, 'put').returns(Promise.resolve());
+      const expectedDoc = {
+        _id: 'connected-user-' + 'userXYZ',
```

Comment:
I think you may have had this more dynamic and changed it but now the concatenation isn't doing anything.

```suggestion
        _id: 'connected-user-userXYZ',
```

Reply:
This conversation was resolved but the code change wasn't made?